tutorials/hyperspectral/RasterStack-RGB-Images-in-R-Using-HSI.Rmd

```diff
@@ -301,15 +276,15 @@ We can play with the color ramps too if we want:
 
 #change the colors of our raster 
 myCol=terrain.colors(25)
```

Comment:
Change assignment operator from = to <- (with spaces on each side)

Reply:
Corrected here and in a few other places.